.github/workflows/integrate.yaml

```diff
@@ -0,0 +1,50 @@
+# https://docs.github.com/en/actions
+
+name: "Integrate"
+
+on:
+  pull_request: null
+  push:
+    branches:
+      - "master"
+
+env:
+  LANGUAGE: "de"
+
+jobs:
+  build:
+    name: "Build"
+
+    runs-on: "ubuntu-latest"
+
+    strategy:
+      matrix:
+        php-version:
+          - "8.0"
```

Comment:
A bit out of scope of the announced change, I adjusted to use PHP 8.0 instead of PHP 7.2 here, to bring this in line with the changes in https://github.com/php/doc-en/pull/299, where @Girgias [suggested](https://github.com/php/doc-en/pull/299#discussion_r550345473) to use PHP 8.0.

Reply:
Yes, that's fine. Thanks!